### ISSUE TITLE
Test the project designs against the relevant tools for Accessibility compliance

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/311-data#1110]
### Dependencies:
Dependency: v2 needs to be completed.

https://github.com/hackforla/311-data/projects/10?card_filter_query=milestone%3A%22initial+v2+launch%22

### Overview
There are more than 100 accessibility testing tools.  Figuring out which ones to use can be a black hole.  For guidance we recommend this article:  [Which accessibility testing tool should you use?](https://medium.com/pulsar/which-accessibility-testing-tool-should-you-use-e5990e6ef0a)

### Summary of Article
The author recommends using the tools in the following order fixing as you go along, since no one tool catches all the relevant issues

[aXe](https://www.deque.com/axe/)
[SiteImprove](https://chrome.google.com/webstore/detail/siteimprove-accessibility/efcfolpjihicnikpmhnmphjhhpiclljc)
[Tenon](https://tenon.io/)
[WAVE](https://wave.webaim.org/extension/)
[Lighthouse](https://developers.google.com/web/tools/lighthouse/#devtools)

But if you want to test your site with other tools, here is a bigger [list](https://www.w3.org/WAI/ER/tools/)

### Action Items
Test your project designs against the relevant tools

**Comment by EchoProject** [comment]
Get an update from UI/UX on accessibility testing. Consider how we approach testing for future iterations.
